lib/pbio/drv/bluetooth/bluetooth_stm32_cc2640.c

```diff
@@ -1094,12 +1095,17 @@ static void handle_event(uint8_t *packet) {
 
             switch (event_code) {
                 case ATT_EVENT_EXCHANGE_MTU_REQ: {
-                    // uint16_t client_mtu = (data[7] << 8) | data[6];
-                    attExchangeMTURsp_t rsp;
+                    uint16_t client_mtu = (data[7] << 8) | data[6];
+
+                    // REVISIT: Just saving the main connection MTU for now.
+                    // If we allow multiple connections, this will need to be
+                    // changed.
+                    if (connection_handle == conn_handle) {
+                        conn_mtu = MIN(client_mtu, PBDRV_BLUETOOTH_MAX_MTU_SIZE);
+                    }
 
+                    attExchangeMTURsp_t rsp;
                     rsp.serverRxMTU = PBDRV_BLUETOOTH_MAX_MTU_SIZE;
```

Comment:
Just curious, why does this response not use the `conn_mtu` value?

Reply:
Per the Bluetooth spec, both the central and peripheral send their own value and take the lesser of the two. This way, it doesn't matter who asks first.

Reply:
Thanks! If you have time at some point, I have a few more Bluetooth questions 😄